crates/oxc_linter/src/snapshots/aria_activedescendant_has_tabindex.snap

```diff
@@ -0,0 +1,19 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: aria_activedescendant_has_tabindex
+---
+  ⚠ eslint-plugin-jsx-a11y(aria-activedescendant-has-tabindex): Enforce elements with aria-activedescendant are tabbable.
+   ╭─[aria_activedescendant_has_tabindex.tsx:1:1]
+ 1 │ <div aria-activedescendant={someID} />;
+   · ──────────────────────────────────────
```

Comment:
can we point just to the `div`?

Reply:
Thank you!
I fixed it to highlight only the `div`.
https://github.com/oxc-project/oxc/pull/2012/commits/de5111e32ebba7dc43cc097588bb97afb20b2b73